.github/workflows/integration-tests.yaml

```diff
@@ -1,25 +1,59 @@
 name: Run Integration Tests
 
 on:
+  repository_dispatch:
+    types: [json-change-detected]  # Custom event type to trigger the workflow
```

Comment:
nit: is it possible to make this more specific?

Reply:
how specific are you thinking? I think we can though, it's all custom

Reply:
maybe something like `test-txn-json-change-detected`?